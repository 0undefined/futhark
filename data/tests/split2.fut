fun {[real], [int]} main(int n, [real] a1, [int] a2) =
  let b = zip(a1,a2) in
  let {first, rest} = split( (n), b) in
  unzip(first)
