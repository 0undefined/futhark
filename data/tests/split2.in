4
[100.0,9.0,1.0,65.0,3.14,6.0,2.0,1.0]
[1,2,6,3,65,1,9,100]