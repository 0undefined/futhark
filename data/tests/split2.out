[100.000000, 9.000000, 1.000000, 65.000000]
[1, 2, 6, 3]
